contributing.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to AVA
+
+✨ Thanks for contributing to AVA! ✨
+
+## How can I contribute?
+
+### Improve documentation
+
+As a user of AVA you're the perfect candidate to help us improve our documentation. Typo corrections, error fixes, better explanations, more examples, etc. Open issues for things that could be improved. [Help translate our docs.](https://github.com/sindresorhus/ava-docs) Anything. Even improvements to this document.
+
+### Improve issues
+
+Some issues are created with missing information, not reproducible, or plain invalid. Help make them easier to resolve. Handling issues takes a lot of time that we could rather spend on fixing bugs and adding features.
+
+### Give feedback on issues
+
+We're always looking for more opinions on discussions in the issue tracker. It's a good opportunity to influence the future direction of AVA.
+
+### Hang out in our chat
+
+We have a [chat](https://gitter.im/sindresorhus/ava). Jump in there and lurk, talk to us, and help others.
+
+### Submitting an issue
+
+- The issue tracker is for issues. Use our [chat](https://gitter.im/sindresorhus/ava) or [Stack Overflow](https://stackoverflow.com/questions/tagged/ava) for support.
+- Search the issue tracker before opening an issue.
+- Ensure you're using the latest version of AVA.
+- Use a clear and descriptive title.
+- Include as much information as possible: Steps to reproduce the issue, error message, Node.js version, operating system, etc.
+- The more time you put into an issue, the more we will.
+- [The best issue report is a failing test proving it.](https://twitter.com/sindresorhus/status/579306280495357953)
+
+### Submitting a pull request
+
+- Non-trivial changes are often best discussed in an issue first.
```

Comment:
Should we add a "why" to this? Something like `to prevent you from doing unnecessary work`.